MIP/mip-73/README.md

```diff
@@ -0,0 +1,77 @@
+# MIP-73: The SanFran Model
+- **Description**: Proposes an L2 model that improves the Biarritz model. It transitions to a Lock/Mint-Native Bridge, adds Bridge fees, and proposes changes to existing components.
+- **Authors**: [Andreas Penzkofer](mailto:andreas.penzkofer@movementlabs.xyz)
+
+## Abstract
+
+The SanFran Model applies the following changes
+
+- formally adds the Lock/Mint-Native Bridge, see [MIP-58](https://github.com/movementlabsxyz/MIP/pull/58)
+- proposes detailed Bridge fees, see [MD-69](https://github.com/movementlabsxyz/MIP/pull/69)
+- proposes Relayer - Continuous Operation and Bootstrapping algorithm, see [MIP-61](https://github.com/movementlabsxyz/MIP/pull/61)
+- proposes changes to the Informer, see [MIP-71](https://github.com/movementlabsxyz/MIP/pull/71)
+- proposes changes to the Rate Limiter, see [MIP-74](https://github.com/movementlabsxyz/MIP/pull/74)
+- proposes fast-confirmations on L2, see [MIP-65](https://github.com/movementlabsxyz/MIP/pull/65)
+
+## Motivation
+
+This model is the result of the co-location.
```

Comment:
Are there any other motivations, such as differences between this and a previous model, that can be mentioned here?